### ISSUE TITLE
[Sugestion] Colored Dynamic Edge Light and Simple Light Bulb

### ISSUE THREAD

**Issue by shiftthedev** [Flanks255/simplylight#43]
As the title says, would be nice if those lights, or even all lights, had colored versions.

**Comment by Veyd3mo** [comment]
I agree with this guy! was so disappointed when i couldnt change the color of them for my cyberpunk build D:

**Comment by shiftthedev** [comment]
I made a custom 1.18.2 version for myself with colored versions, if you want I can make a pull request tomorow.

**Comment by Flanks255** [comment]
the point wasn't if i can, its if i should.

**Comment by Flanks255** [comment]
I have been tossing around the idea of making a official add-on mod that would add lights like this, and others in the future, that dont really fit the spirit of the base mod(in this case by adding a ton of extra blockstates).

**Comment by Somnioblivio** [comment]
> I have been tossing around the idea of making a official add-on mod that would add lights like this, and others in the future, that dont really fit the spirit of the base mod(in this case by adding a ton of extra blockstates).

Simply light for me is just that... a lights mod... simple. 

All of the white lights are nice but man, they stick out hard on most builds but if i could color them in to different colors by say whacking them with a dye so that they glow a different color i could see myself using them constantly... please please please add this.

**Comment by TetyLike3** [comment]
I know the mod name is Simply Light and all... but it would be nice to have a light type that changes colour upon redstone input (maybe as an addon like you mentioned). I'm building a factory and it would be awesome to have the lights go red when I flick the alarm switch.

**Comment by Flanks255** [comment]
Perhaps if i make some kind of programmable indicator or something.

At the moment i have the colored versions of all the other lights about 3/4 done, i just am taking a break to work on porting other mods before i finish it, theres other features i want to release at the same time lol.

> I know the mod name is Simply Light and all... but it would be nice to have a light type that changes colour upon redstone input (maybe as an addon like you mentioned). I'm building a factory and it would be awesome to have the lights go red when I flick the alarm switch.

**Comment by Flanks255** [comment]
Added in 1.5.0
